packages/office-addin-lint/src/lint.ts

```diff
@@ -59,12 +67,16 @@ export function getPrettierCommand(files: string): string {
 }
 
 export function makeFilesPrettier(files: string) {
-  try{
+  try {
     const command = getPrettierCommand(files);
     execCommand(command);
     usageDataObject.sendUsageDataSuccessEvent("makeFilesPrettier");
```

Comment:
Same goes here. Need to better clarify when the prettier command succeed or has an exception, and include the status code with the success event. (If an exception is thrown, I would suspect the status code wouldn't apply and the err would contain the info.)